email_template_qweb/models/mail_template.py

```diff
@@ -1,19 +1,51 @@
-# Copyright 2016 Therp BV <http://therp.nl>
+# Copyright 2016,2025 Therp BV <http://therp.nl>
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
-from odoo import fields, models, tools
+from odoo import _, api, fields, models, tools
 
 
 class MailTemplate(models.Model):
     _inherit = "mail.template"
 
+    def _get_edit_language_selection(self):
+        """Selection for language to edit template."""
+        active_languages = self.env["res.lang"].search(
+            [("active", "=", True), ("code", "!=", "en_US")]
+        )
+        selection = [("en_US", "English (US)")] + [
+            (lang.code, lang.name) for lang in active_languages
+        ]
+        return selection
+
+    @api.depends("edit_language", "body_view_id.arch")
+    def _compute_body_view_arch(self):
+        for this in self:
+            this.body_view_arch = this.body_view_id.with_context(
+                lang=this.edit_language
+            ).arch
+
+    def _inverse_body_view_arch(self):
+        for this in self:
+            this.body_view_id.with_context(
+                lang=this.edit_language
+            ).arch = this.body_view_arch
+
```

Comment:
those go below the field definitions, cf https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#37models